readme.md

```diff
@@ -29,7 +27,7 @@ taskkill(input, function (err) {
 
 See the [`taskkill` docs](https://technet.microsoft.com/en-us/library/bb491009.aspx) for more.
 
-### taskkill(input, [options], [callback])
+### taskkill(input, [options])
```

Comment:
You're right. Not sure if Sindre is putting a return description if a promise returns nothing. It might be a nice addition though. The only way you know it's returning a promise is by looking at the example.